content/courses/ada_2022_whats_new/chapters/image_attribute.rst

```diff
@@ -1,20 +1,116 @@
 .. include:: ../../global.txt
 
-:ada:`'Image` attribute
-=======================
+:ada:`'Image` attribute for any type
+====================================
+
+.. note::
+
+   Attribute :ada:`'Image` for any type is supported by
+
+    * GNAT Community Edition 2020 and latter
+    * GCC 11
+
+:ada:`'Image` attribute for a value
+-----------------------------------
+
+If you missed `Technical Corrigendum 1`_ changes (they were published
+in February 2016) then you probably don't know that the :ada:`'Image`
+attribute can be applied now to a value. So, instead of
```

Comment:
Since our intent here is just to teach, I'd rather avoid something like "If you missed..." Instead, you could write this for example:

```suggestion
With the publication of the `Technical Corrigendum 1`_ in February
2016, the :ada:`'Image`attribute can now be applied to a value. So, 
instead of
```